src/containers/Header.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+
```

Comment:
```suggestion
import Menu from "./Menu";
```

используете компонент `Menu`, но не импортируете